src/Components/Textfield/Textfield.js

```diff
@@ -36,15 +36,14 @@ const Textfield = ({ componentDetails, data, handleTextfieldChange, index, submi
       helperText={errorController.showError && errorController.message(data[inputName])}
       inputProps={numericField ? { pattern: '[0-9]* .', inputMode: 'decimal' } : {}}
       InputProps={
-        (dollarField
```

Comment:
If we add `...` in the front, we don't need to pass in `sx: { backgroundColor: '#FFFFFF' }` twice.